docs/sources/source/nlp/joint_intent_slot_filling.rst

```diff
@@ -6,12 +6,12 @@ In this tutorial, we are going to implement a joint intent and slot filling syst
 All code used in this tutorial is based on ``examples/nlp/intent_detection_slot_tagging/joint_intent_slot_with_bert.py``.
 
 There are a variety pre-trained BERT models that we can select from using the argument `--pretrained_bert_model`. We're currently
-using the script for loading pre-trained models from `transformers`. See the list of available pre-trained models
-`here <https://https://huggingface.co/transformers/pretrained_models.html>`__.
+using the script for loading pre-trained models from `transformers`. \
+See the list of available pre-trained models by calling `nemo.collections.nlp.nm.trainables.get_bert_models_list()`.
 
 .. tip::
```

Comment:
Oh, I did not know that. What changes should I do? Just the dataset?

Reply:
In the example script you'd need to initialize a Nemo tokenizer and then when the tokenizer is used (dataset/data descriptor) use methods supported be the new tokenizers, for example, `text_to_tokens()` instead of `tokenize()`

Reply:
Changed to the new style. Please check it out.

Reply:
Updated inference scripts too.